scripts/repl.js

```diff
@@ -57,6 +62,86 @@
   };
 
   /*
+   * Utils for working with parse.com for persistence
+   */
+  function ParseUtils () {}
+
+  var __parseLoadingCallbacks = null;
```

Comment:
Not currently, but it'd be fragile.

```js
new ParseUtils().load().afterLoad(f);
new ParseUtils().load().afterLoad(g);
```

This loads parse once, both f and g are called when it's loaded.  Changing to to an instance property would load parse twice and maybe break things.  It could be made a property of ParseUtils if that's prefered.